Realm.Shared/linq/RealmResults.cs

```diff
@@ -148,11 +148,16 @@ IEnumerator IEnumerable.GetEnumerator()
 
 
         /// <summary>
-        /// Fast count all objects of a given class.
+        /// Fast count all objects of a given class, or in a RealmResults after casting.
         /// </summary>
         /// <remarks>
         /// Resolves to this method instead of the LINQ static extension <c>Count&lt;T&gt;(this IEnumerable&lt;T&gt;)</c>, when used directly on Realm.All.
+		/// <br>
+		/// if someone CASTS a RealmResults<blah> variable from a Where call to 
```

Comment:
Should this be `RealmResults&lt;T&gt;`? Or does the `remarks` tag properly handle those? Even if that's the case, we probably should replace `<blah>` with `<T>`